.rubocop.yml

```diff
@@ -5,7 +5,7 @@ AllCops:
     - 'test/**/**/*'
     - 'bin/*'
 
-Style/SpaceAfterControlKeyword:
+Style/SpaceAroundKeyword:
```

Comment:
When I ran rubocop locally, it complained that SpaceAfterControlKeyward was deprecated.